app/controllers/users_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
   end
 
   def show
+    @ban_json = JSON.parse(File.read("/etc/minecraft/redstoner/banned-players.json")).detect {|u| u["uuid"] == @user.uuid}
```

Comment:
This file can be quite huge, so it would be better to cache the parsed file (in the User model) for 10s or the likes